bundles/org.openhab.binding.asuswrt/README.md

```diff
@@ -0,0 +1,103 @@
+|                  | memFree            | Number:DataAmountype   | Free memory in MB                          | router                            |
+|                  | memUsedPercent     | Number:Dimensionles    | Used memory in %                           | router                            |
+|                  | cpuUsedPercent     | Number:Dimensionles    | Total CPU usage in percent over all cores  | router                            |
+| clientListGroup  | knownClients       | text (RO)              | Known clients with name and MAC-Addresses  | router                            |
+|                  | onlineClients      | text (RO)              | Online clients with name and MAC-Addresses | router                            |
+|                  | onlineMACs         | text (RO)              | List with mac-addresses of online clients  | router                            |    
+|                  | onlineClientsCount | Number:Dimensionless   | Count of online clients                    | router                            |
+| traffic          | curRX              | Number:DataTransferRate| Current DataTransferRate MBits/s (receive) | interface, client                 |
+|                  | curTX              | Number:DataTransferRate| Current DataTransferRate MBits/s (send)    | interface, client                 |
+|                  | todayRX            | Number:DataAmount      | Data received since 0:00 a clock in MB     | interface, client                 |
+|                  | todayTX            | Number:DataAmount      | Data sent since 0:00 a clock in MB         | interface, client                 |
+|                  | totalRX            | Number:DataAmount      | Data received since reboot in MB           | interface, client                 |
+|                  | totalTX            | Number:DataAmount      | Data sent since reboot in MB               | interface, client                 |
+
+
+## Events
+
+All devices support some of the following Events:
+| group            | event             |kind        | description                                                            | things supporting this event    |
+|------------------|-------------------|------------|------------------------------------------------------------------------|---------------------------------|
+| networkInfo      | connectionEvent   | Trigger    | Fired if connection is established ('connected') or ('disconnected')   | interface                       |
+|                  | clientOnlineEvent | Trigger    | Fired if Client leaves ('gone') or enters ('connected') the network    | client                          |
+| clientListGroup  | clientOnlineEvent | Trigger    | Fired if Client leaves ('gone') or enters ('connected') the network    | router                          |
+
+
+
+## Debugging and Tracing
+
+If you want to see what's going on in the binding, switch the loglevel to TRACE in the Karaf console
```

Comment:
I don't think this should be documented here as it is duplicate and allready documented here: https://www.openhab.org/docs/administration/logging.html#defining-what-to-log